### ISSUE TITLE
How to load custom Yolov5 model

### ISSUE THREAD

**Issue by shima97** [mikel-brostrom/Yolov5_StrongSORT_OSNet#505]
### Search before asking

- [X] I have searched the Yolov5_StrongSORT_OSNet [issues](https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/issues) and found no similar bug report.

### Question

Dear Mr.  Brostrom
 I have a problem changing the weight of the detection part from deepSORT. 
It is very kind of you to tell me how I can change the default weight with my own retrained YOLOV5 weight for the object detection part. That is needed because of poor object detection and lost of objects with default weights of deepSORT.

**Comment by mikel-brostrom** [comment]
```python track.py --source 0 --yolo-weights /path/to/your/yolov5.pt```

**Comment by sSlotty** [comment]
I'm having the same problem
Error message `AttributeError: Can't get attribute 'DetectionModel'`

**Comment by sssssshf** [comment]
> `python track.py --yolo-weights /path/to/your/yolov5.pt`

how to load myself strong_sort_model??????

**Comment by mikel-brostrom** [comment]
> how to load myself strong_sort_model??????

```bash
python track.py --strong-sort-weights /path/to/your/strong/sort/weights.pt
```

**Comment by sssssshf** [comment]
> > 如何加载自己strong_sort_model？？？？？？
> 
> ```shell
> python track.py --strong-sort-weights /path/to/your/strong/sort/weights.pt
> ```

how to plot the reid's Trajectory?

**Comment by mikel-brostrom** [comment]
You have to implement this yourself. This functionality does not exist in the repo

**Comment by shahzaib3120** [comment]
> I'm having the same problem Error message `AttributeError: Can't get attribute 'DetectionModel'`

Same here, any idea how to make it work? I trained my detection model on yolov5 from scratch with a single class and it is working with good confidence.
Do I need to retrain deepSort as well?

I am running this command.

`python track.py --source video/my_video.mp4 --strong-sort-weights osnet_x0_25_market1501.pt --yolo-weights weights/my_yolov5_weights.pt --classes 0 --save-vid --show-vid `

I have also tried running without `--classes 0`. getting the same error.

Thanks in anticipation.
